openedx/core/djangoapps/credit/api.py

```diff
@@ -322,6 +322,93 @@ def get_credit_requests_for_user(username):
     return CreditRequest.credit_requests_for_user(username)
```

Comment:
@srpearce When you have a moment, let's discuss this offline.  The answer is somewhat complex and easier to discuss in person :)